morebits.js

```diff
@@ -1398,6 +1398,7 @@ Morebits.wiki.api = function( currentAction, query, onSuccess, statusElement, on
 	this.currentAction = currentAction;
 	this.query = query;
 	this.query.format = 'xml';
+	this.query.assert = 'user';
```

Comment:
@atlight That worked. I thought I could put it here but didn't take the time to try =P